src/draw/draw3d.jl

```diff
@@ -0,0 +1,87 @@
+#
+# This file is a part of MolecularGraph.jl
+# Licensed under the MIT License http://opensource.org/licenses/MIT
+#
+
+# This is loaded via @require
+
+using .AbstractPlotting
+
+export spacefilling, ballstick
+
+atomcolors = Dict("O" => colorant"red",
+                  "C" => colorant"black",
+                  "H" => colorant"lightgray",
+                  "S" => colorant"yellow",
+                  "P" => colorant"green")
```

Comment:
Happy to add more colors to this list. Just let me know what you want.

Reply:
How about `"N" => colorant"blue",`?

Here is the colors in 2D depiction
https://github.com/mojaie/MolecularGraph.jl/blob/aef2b1d4f02e508256d85e54392d97b5b45a97fa/src/draw/draw2d.jl#L35-L49

Reply:
I incorporated these. One oddity is `:H` being black: that seems different from CPK coloring, https://en.wikipedia.org/wiki/CPK_coloring.

Reply:
Yes,  :H has been black for visibility on white background, but gray would be a good alternative. I want to consider making atom color set options from several CPK colors.